hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -145,10 +145,10 @@ <h4>Extent</h4>
 
     {# Add THREDDS service #}
     {% if nc_file_name %}
-        <div id="#thredds">
-            <h3>THREDDS Data Service</h3>
-            <h4>OPeNDAP: <a href="{{ opendap }}" target="_blank"> {{ nc_file_name }} </a></h4>
-            <h4>Visualize using IDV:Requires IDV <a href="https://www.unidata.ucar.edu/software/idv/" target="_blank">installation</a>, then use OPeNDAP service link as "URL" for IDV "Data Choosers"</h4>
+        <div id="#thredds" class="col-sm-12 space-top">
```

Comment:
I will be reverting this part, looks like some unstashed changes from one of my other branches.